pkg/pillar/cmd/zedrouter/zedrouter.go

```diff
@@ -1033,13 +1009,21 @@ var deviceEID net.IP
 var deviceIID uint32
 var additionalInfoDevice *types.AdditionalInfoDevice
 
-func handleAppNetworkCreate(ctx *zedrouterContext, key string,
-	config types.AppNetworkConfig) {
+func handleAppNetworkCreate(ctxArg interface{}, key string, configArg interface{}) {
+
+	ctx := ctxArg.(*zedrouterContext)
+	config := cast.CastAppNetworkConfig(configArg)
 	log.Infof("handleAppAppNetworkCreate(%v) for %s\n",
 		config.UUIDandVersion, config.DisplayName)
 
 	// Pick a local number to identify the application instance
```

Comment:
The comment is now misplaced relative to its code.
FWIW there was a line of comment for the receivedConfigTime code snipped, and it might have made sense to move that one here.